packages/ic-management/src/types/ic-management.params.ts

```diff
@@ -132,3 +132,5 @@ export interface ProvisionalTopUpCanisterParams {
   canisterId: Principal;
   amount: bigint;
 }
+
+export type SnapshotIdText = string;
```

Comment:
this type is not exported in `index`?

Reply:
In `index`, this all module, `ic-management.params` is exposed.

```
...
export * from "./types/ic-management.params";
...
```

Reply:
perfect tks!